src/mmrelay/matrix_utils.py

```
@@ -440,46 +496,64 @@ def _canonical_server(value: str | None) -> str | None:
     return f"@{localpart.rstrip(':')}:{domain}"
 
 
-def _get_msgs_to_keep_config():
+def _get_msgs_to_keep_config(config_override: dict[str, Any] | None = None) -> int:
     """
-    Determine how many Meshtastic–Matrix message mappings should be retained.
-
-    Prefers the new configuration path `database.msg_map.msgs_to_keep`. If that path is absent, falls back to the legacy `db.msg_map.msgs_to_keep` and emits a deprecation warning. If no configuration or value is present, returns DEFAULT_MSGS_TO_KEEP.
-
+    Determine how many Meshtastic–Matrix message mappings to retain.
```

Ruff (0.14.10)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)